### PR TITLE
chore: Remove feature-flag for git

### PR DESCRIPTION
**Pull request by abhvsn**
## Description

> PR to remove feature flag for git APIs, we are still keeping the git-import under the feature flag as this is still in progress

### CODE REVIEW

File: app/server/appsmith-server/src/main/java/com/appsmith/server/exceptions/AppsmithError.java [hunk 1]
```diff
@@ -131,8 +131,8 @@
...
     GIT_EXECUTION_TIMEOUT(504, 5014, "Git command execution exceeded the maximum allowed time, please contact Appsmith support for more details", AppsmithErrorAction.DEFAULT, null, ErrorType.CONNECTIVITY_ERROR, null),
     INCOMPATIBLE_IMPORTED_JSON(400, 4045, "Provided file is incompatible, please upgrade your instance to resolve this conflict.", AppsmithErrorAction.DEFAULT, null, ErrorType.BAD_REQUEST, null),
-    GIT_MERGE_CONFLICTS(400, 4046, "Merge conflicts found: {1}", AppsmithErrorAction.DEFAULT, null, ErrorType.GIT_ACTION_EXECUTION_ERROR, ErrorReferenceDocUrl.GIT_MERGE_CONFLICT),
-    GIT_PULL_CONFLICTS(400, 4047, "Merge conflicts found during the pull operation: {1}", AppsmithErrorAction.DEFAULT, null, ErrorType.GIT_ACTION_EXECUTION_ERROR, ErrorReferenceDocUrl.GIT_MERGE_CONFLICT),
+    GIT_MERGE_CONFLICTS(400, 4046, "Merge conflicts found: {0}", AppsmithErrorAction.DEFAULT, null, ErrorType.GIT_ACTION_EXECUTION_ERROR, ErrorReferenceDocUrl.GIT_MERGE_CONFLICT),
+    GIT_PULL_CONFLICTS(400, 4047, "Merge conflicts found during the pull operation: {0}", AppsmithErrorAction.DEFAULT, null, ErrorType.GIT_ACTION_EXECUTION_ERROR, ErrorReferenceDocUrl.GIT_PULL_CONFLICT),
```
Comment (abhvsn) [inline app/server/appsmith-server/src/main/java/com/appsmith/server/exceptions/AppsmithError.java:135]: Small fix to respond with the correct doc url

Comment (abhvsn) [inline app/server/appsmith-server/src/main/java/com/appsmith/server/exceptions/AppsmithError.java:135]: Small fix to respond with the correct doc URL for pull conflicts

Comment (abhvsn) [inline app/server/appsmith-server/src/main/java/com/appsmith/server/exceptions/AppsmithError.java:135]: Small fix to respond with the correct doc URL for pull conflicts

Comment (abhvsn) [inline app/server/appsmith-server/src/main/java/com/appsmith/server/exceptions/AppsmithError.java:135]: Small fix to respond with the correct doc URL for pull conflicts

**Review by AnaghHegde — APPROVED** [review]

**Pull request merged**